src/data-tutorials/users.tsx

```diff
@@ -695,6 +695,15 @@ const Users: User[] = [
     website: '/blog/tutorials/securing-multi-sig-wallet-using-hardware-wallet-the-benefits-for-securing-community-funds',
     tags: ['beginner','celosage','celo'],
   },
+  
+  
+    {
+    title: "How to Store NFT Smart Contract Assets Using The FileBase Decentralized Storage System",
+    description: "This article gives a comprehensive overview of decentralized storage solutions. It contains a clear walk-through of how to store digital assets on the filebase storage solution.",
+    preview: require('./showcase/intermediate/filebase-cover.png'),
+    website: '/tutorials/how-to-store-NFT-smart-contract-assets-using-the-fileBase-decentralized-storage-system',
+    tags: ['intermediate', 'solidity', 'smart contract', 'celosage', 'celo', 'filebase', 'decentralized storage'],
```

Comment:
the tag "smart contract" shouldn't have a space in between It should be "smartcontract" same with "decentralizedstorage"